index.js

```diff
@@ -92,6 +92,7 @@ type Options = {
   failOnCancel?: boolean,
   showAppsToView?: boolean,
   saveToFiles?: boolean,
+  appId?: string,
```

Comment:
```suggestion
  appId: string,
```

This should be required, no?